guarddog/cli.py

```diff
@@ -195,23 +266,34 @@ def _scan(identifier, version, rules, exclude_rules, output_format, exit_non_zer
         sys.exit(1)
     results = []
     if is_local_target(identifier):
-        log.debug(f"Considering that '{identifier}' is a local target, scanning filesystem")
+        log.debug(
+            f"Considering that '{identifier}' is a local target, scanning filesystem"
+        )
         if os.path.isdir(identifier):
             log.debug(f"Considering that '{identifier}' as a local directory")
             for package in os.listdir(identifier):
-                results.append({'package': package} | scanner.scan_local(f"{identifier}/{package}", rule_param))
+                results.append(
```

Comment:
in terms of readability, IMHO having this split in 2 would be slightly more readable (same for the "else" statement) 

```
result = scanner.scan_local(f"{identifier}/{package}", rule_param)
result["package"] = package
results.append(result)
```

Reply:
done in 79242a5ff004eaf406019de6f3cd7ece95052489